recipes/quantization.md

```diff
@@ -1,14 +1,11 @@
 # Quantization and Sparsity
 
-torchtune integrates with [torchao](https://github.com/pytorch/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
+torchtune integrates with [torchao](https://github.com/pytorch/ao/) for QAT and QLoRA. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
 
-#### Quantize
-To quantize a model (default is int4 weight only quantization):
-```
-tune run quantize --config quantization
```

Comment:
note: this is still referenced in the QAT tutorial, so maybe we need to keep this in README but move it to under the QAT section?